### PR TITLE
Fix: Template is not importing when the Unfiltered File option is inactive [ED-13682]

### PR DESCRIPTION
**Pull request by mark-elementor**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: assets/dev/js/admin/admin.js [hunk 1]
```diff
@@ -407,14 +407,15 @@ import { showJsonUploadWarningMessageIfNeeded } from 'elementor-utils/json-uploa
 				$( '#elementor-import-template-area' ).toggle();
 			} );
 
-			let messageShown = false;
+			let warningMessageShown = false;
+			let notEnabledMessageShown = false;
```
Comment (ManorHazaz) [inline assets/dev/js/admin/admin.js:411]: Not clear what it represents maybe a different name?

File: assets/dev/js/admin/admin.js [hunk 2]
```diff
@@ -434,10 +435,11 @@ import { showJsonUploadWarningMessageIfNeeded } from 'elementor-utils/json-uploa
 					return;
 				}
 
-				if ( $importFormFileInput[ 0 ].files.length && ! elementorCommon.config.filesUpload.unfilteredFiles ) {
+				if ( $importFormFileInput[ 0 ].files.length && ! elementorCommon.config.filesUpload.unfilteredFiles && ! notEnabledMessageShown ) {
```
Comment (ManorHazaz) [inline assets/dev/js/admin/admin.js:438]: It's getting complicated maybe use helper variables for the other conditions?

**Review by ManorHazaz — CHANGES_REQUESTED** [review]

File: assets/dev/js/admin/admin.js [hunk 3]
```diff
@@ -434,10 +435,11 @@ import { showJsonUploadWarningMessageIfNeeded } from 'elementor-utils/json-uploa
...
 					event.preventDefault();
 
 					const enableUnfilteredFilesModal = FilesUploadHandler.getUnfilteredFilesNotEnabledImportTemplateDialog( () => {
+						notEnabledMessageShown = true;
 						$importForm.trigger( 'submit' );
```
Comment (Omerisra6) [inline assets/dev/js/admin/admin.js:443]: I'm not sure why we need to re trigger the form submit if it is already submitted.

File: assets/dev/js/admin/admin.js (hunk 1 above)
Comment (mark-elementor) [reply assets/dev/js/admin/admin.js]: Done.

File: assets/dev/js/admin/admin.js (hunk 3 above)
Comment (mark-elementor) [reply assets/dev/js/admin/admin.js:449]: We sort-of re-submit after the user approved all warnings.

**Review by ManorHazaz — APPROVED** [review]

**Review by Omerisra6 — APPROVED** [review]

**Pull request merged**